docs/user-guide/features.rst

```diff
@@ -33,6 +33,27 @@ Administrators can also :ref:`upload <create-errata-recipe>` their own errata to
 a repository. Please see the :doc:`recipes` documentation to learn how to
 perform these operations.
 
+Boolean dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Pulp supports RPM content with `boolean dependencies
+<http://rpm.org/user_doc/boolean_dependencies.html>`_ in the ``Requires`` field
+in these basic repository and content management use cases:
+
+* synchronization, publication and content upload
+
+* copying content between repositories
+
+* displaying boolean dependencies in the ``Requires`` field
+
+* providing the content to the ``dnf`` client to process boolean dependencies
+
+.. Note::
+
+  Pulp actually processing boolean dependencies is a matter of future features.
+  A recursive copy won't therefore work properly right now for content that has
```

Comment:
Can you add link to a recursive copy docs or example? I think we had one, just to make sure everyone understands what is not supported yet.

Reply:
Maybe: `recursive copy is not supported yet for content that has ...`?

Reply:
recipes contain those multiple times but all the time with specific content type; lemme update with an in-line example for clarity